src/include/parser/explain_statement.h

```diff
@@ -0,0 +1,40 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// explain_statement.h
+//
+// Identification: src/include/parser/explain_statement.h
+//
+// Copyright (c) 2018, Pranav Kant
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include "common/logger.h"
+#include "common/sql_node_visitor.h"
+#include "expression/abstract_expression.h"
+#include "parser/sql_statement.h"
+#include "parser/table_ref.h"
+
+namespace peloton {
+namespace parser {
+
+/**
+ * @class ExplainStatement
+ * @brief Represents "EXPLAIN <query>"
+ */
+class ExplainStatement : public SQLStatement {
+ public:
+  ExplainStatement()
+      : SQLStatement(StatementType::EXPLAIN) {}
+  virtual ~ExplainStatement() {}
+
+  virtual void Accept(SqlNodeVisitor* /*v*/) override { }
```

Comment:
Remove `virtual` keyword.

Reply:
This is consistent with other *_statement.h files. So, I didn't want to deviate unless you have a strong reason to do so?

Reply:
The `virtual` keyword is unnecessary because we have `override`. Ideally, all other statement classes should change, but we'll update them incrementally.